Explorer/Assets/DCL/PlacesAPIService/PlacesAPIService.cs

```diff
@@ -93,8 +93,7 @@ public async UniTask<List<PlacesData.PlaceInfo>> GetPlacesByCoordsListAsync(IEnu
             }
 
             places.AddRange(alreadyCachedPlaces);
-
-            return places;
+            return rentedPlaces;
         }
 
         public async UniTask<IReadOnlyList<PlacesData.PlaceInfo>> GetFavoritesAsync(int pageNumber, int pageSize, CancellationToken ct, bool renewCache = false)

```

Comment:
I would like you to pay attention to how you use this method as it returns the reusable list. If it's called several times and the result is saved somewhere then it will be a failure. 
For safety you could you the same approach with `PoolExtensions.Scope<List<PlacesData.PlaceInfo>`